package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "mss-harbor",
-  "version": "1.10.2",
+  "name": "harbor-ui",
+  "version": "1.11.0",
   "description": "A portal for creating and viewing Shipments",
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "postinstall": "grunt build",
-    "test": "grunt test"
+    "rebuild": "./node_modules/grunt-cli/bin/grunt build",
```

Comment:
I'm confused by your first two sentences. 

However, you should be able to name it `build` or `rebuild` and it doesn't matter if it's a `npm` command (or not), because you'll run it as `npm run build` or `npm run rebuild`. 

FWIW, `npm start` and `npm test` are special in that they look at what's in the `scripts` block. All other `script` block commands must be run as `npm run <command>`.

Also, it looks as though your `postinstall` didn't work because you ran it as `npm postinstall` instead of `npm run postinstall`

From the `circle.yml` file in the above link:
```yml
dependencies:
  pre:
    - npm install
    - npm postinstall
    - npm test
```